workflow/scripts/reads_manifest.py

```
@@ -0,0 +1,32 @@
+import sys
+import os
+from pathlib import Path
+import pandas as pd
+
+sys.stderr = sys.stdout = open(snakemake.log[0], "w")
```

Ruff

Use a context manager for opening files

(SIM115)